src/vhost_user/mod.rs

```diff
@@ -83,6 +83,12 @@ pub enum Error {
     FeatureMismatch,
     /// Error from request handler
     ReqHandlerError(IOError),
+    /// memfd file creation error
+    MemFdCreateError,
+    /// File truncate error
+    FileTrucateError,
+    /// memfd file seal errors
+    MemFdSealError,
```

Comment:
Why do these errors need to be defined here if they're going to be used in another crate?

Reply:
IIUC because they are returned by functions that returns `vhost::vhost_user::Result`. @harshanavkis any thoughts?